runtime/drive_handler_test.go

```diff
@@ -95,7 +95,7 @@ func TestContainerStubs(t *testing.T) {
 		mockMachine, err := firecracker.NewMachine(ctx, firecracker.Config{}, firecracker.WithClient(
 			firecracker.NewClient("/path/to/socket", nil, false, firecracker.WithOpsClient(&fctesting.MockClient{
 				PatchGuestDriveByIDFn: func(params *ops.PatchGuestDriveByIDParams) (*ops.PatchGuestDriveByIDNoContent, error) {
-					assert.Equal(t, hostPath, firecracker.StringValue(params.Body.PathOnHost))
+					assert.Equal(t, hostPath, firecracker.StringValue(&params.Body.PathOnHost))
```

Comment:
why the `&` here and on line `243` ?

Reply:
This is because the original definition is changed. As you can find [HERE](https://github.com/firecracker-microvm/firecracker-go-sdk/blob/45eff16db190c59d1dd04d9ffc402b06317a2970/pointer_helpers.go). This definition changed is not made by me. To use the new StringValue(), I have to pass an address.